guides/How To Deploy Python App Using uWSGI And Nginx.md

```diff
@@ -195,12 +122,18 @@ To quit Postgres:
 \q
 ```
 
+Change back to user johndoe
+
+```
+su johndoe
+```
+
```

Comment:
Instead of this, should we use `exit`? https://askubuntu.com/questions/489933/how-to-come-out-of-root-user

Reply:
What if we leave both?
`exit` might be a bit more ambiguous. For example, if the previous user was the root and not johndoe.

Reply:
Another option change the text to `Change to user johndoe` because before this we were using the root user.

Reply:
Updated